app/config/init.js

```diff
@@ -39,6 +40,10 @@ const _init = function (cfg) {
     // Ignore undefined values in plugin and localPlugins array Issue #1862
     _cfg.plugins = (_cfg.plugins && _cfg.plugins.filter(Boolean)) || [];
     _cfg.localPlugins = (_cfg.localPlugins && _cfg.localPlugins.filter(Boolean)) || [];
+    if (!fs.existsSync(_cfg.config.shell)) {
```

Comment:
test this before. No need to add fs there. The script will know it if `_cfg.config.shel` exist before the assignation.